assets/js/apicalls.js

```diff
@@ -0,0 +1,79 @@
+
+
+const omdbAPIkey = "5af38ba4";
+var test = document.getElementById('root');
+
+function apicall() {
+  //placeholder movie + key
+  fetch('https://ghibliapi.herokuapp.com/films')
+  .then(response => {
+    return response.json();
+  })
+  //returns movie data as array
+  .then(data => {
+    console.log(data);
+  });
+}
+
+function initApi() {
+  apicall();
+  buildOMDBApiCall();
+  var request = new XMLHttpRequest();
```

Comment:
Move this code into a fetchAllGhibliMovies() function, which calls the endpoint and returns an array of ghibli movies.